src/SignalR/server/SignalR/test/WebSocketsTransportTests.cs

```diff
@@ -86,7 +86,14 @@ public async Task WebSocketsTransportSendsUserAgent()
                     .Assembly
                     .GetCustomAttribute<AssemblyInformationalVersionAttribute>();
 
-                Assert.Equal("Microsoft.AspNetCore.Http.Connections.Client/" + assemblyVersion.InformationalVersion, userAgent);
+                Assert.StartsWith("Microsoft SignalR/", userAgent);
```

Comment:
I think it's OK to be stricter with this test. If we change the User-Agent format, that could break customers.

```suggestion
                Assert.StartsWith($"Microsoft SignalR/{majorVersion}.{minorVersion} ({assemblyVersion.InformationalVersion}; ", userAgent);
```